.github/workflows/pull_request_release.yml

```
@@ -22,6 +24,9 @@ jobs:
           NODE_ENV=development npm ci
           git config --local user.email "one.amex@aexp.com"
           git config --local user.name "OneAmexBot"
+          npm run release:changelog
```

```suggestion
          echo -e "$(head -3 CHANGELOG.md)\n\n$(node_modules/.bin/conventional-changelog -p angular)\n\n$(tail -n +3 CHANGELOG.md)" > CHANGELOG.md
```
So `conventional-changelog -p angular -i CHANGELOG.md -s` might not be the perfect format. It does write  the changes but it pastes it on top of `# Changelog` rather than in the proper place.